src/app/api/users/[email]/route.ts

```diff
@@ -0,0 +1,23 @@
+import { infoMessages } from "@/data/responseMessages"
+import { db } from "@/lib/db"
+import { dbUser } from "@/model/dbUserModel"
+import { NextResponse, NextRequest } from "next/server"
+
+
+export async function GET(request: NextRequest) {
+    const url = await request.nextUrl.pathname
+    const email = url.split("/").pop()
+
+    console.log(email)
+    try {
+        const [rows] = await db.query(
+            `SELECT * FROM users WHERE email = ?`,
```

Comment:
consider refactor users table in user table and refactor where you use the table